scripts/upgrade-assure/update-genesis.go

```diff
@@ -42,7 +42,7 @@ func updateGenesis(validatorBalance, homePath, genesisFilePath string) {
 	}
 
 	// update supply
-	genesis.AppState.Bank.Supply = genesis.AppState.Bank.Supply.Sub(coinsToRemove...).Add(sdk.NewCoin("uelys", newValidatorBalance)).Add(sdk.NewCoin("ibc/2180E84E20F5679FCC760D8C165B60F42065DEF7F46A72B447CFF1B7DC6C0A65", newValidatorBalance)).Add(sdk.NewCoin("ibc/E2D2F6ADCC68AA3384B2F5DFACCA437923D137C14E86FB8A10207CF3BED0C8D4", newValidatorBalance))
+	genesis.AppState.Bank.Supply = genesis.AppState.Bank.Supply.Sub(coinsToRemove...).Add(sdk.NewCoin("uelys", newValidatorBalance)).Add(sdk.NewCoin("ibc/2180E84E20F5679FCC760D8C165B60F42065DEF7F46A72B447CFF1B7DC6C0A65", newValidatorBalance)).Add(sdk.NewCoin("ibc/E2D2F6ADCC68AA3384B2F5DFACCA437923D137C14E86FB8A10207CF3BED0C8D4", newValidatorBalance)).Add(sdk.NewCoin("ibc/B4314D0E670CB43C88A5DCA09F76E5E812BD831CC2FEC6E434C9E5A9D1F57953", newValidatorBalance))
```

Comment:
Hmmm a bit challenging to keep the pace w/ this file the moment users start adding pools. Something to think later

Reply:
@Wgil this is optional, keep in my upgrade-assure uses the testnet snapshot that means if the pools exists on testnet, they will exist on localnet without any changes in the code of upgrade-assure, that code here allows to feed some tokens to the validator account in case we need to use those tokens from validator account, but one can very well use a testnet account that already has those tokens in localnet.

Reply:
thanks for clarifying